src/helm/benchmark/scenarios/legal_summarization_scenario.py

```diff
@@ -0,0 +1,194 @@
+from pathlib import Path
+
+from typing import List, Optional, Any
+
+import datasets
+from datasets import load_dataset
+
+from .scenario import Scenario, Instance, Reference, TRAIN_SPLIT, VALID_SPLIT, TEST_SPLIT, CORRECT_TAG, Input, Output
+
+_ALL_LANGUAGES = {
+    "bulgarian": "bg",
+    "czech": "cs",
+    "dutch": "nl",
+    "estonian": "et",
+    "french": "fr",
+    "greek": "el",
+    "irish": "ga",
+    "latvian": "lv",
+    "maltese": "mt",
+    "portuguese": "pt",
+    "slovak": "sk",
+    "spanish": "es",
+    "croatian": "hr",
+    "danish": "da",
+    "english": "en",
+    "finnish": "fi",
+    "german": "de",
+    "hungarian": "hu",
+    "italian": "it",
+    "lithuanian": "lt",
+    "polish": "pl",
+    "romanian": "ro",
+    "slovenian": "sl",
+    "swedish": "sv",
+}
+
+# IMPORTANT: So far, we only support English, since we would need to adapt the metrics for scoring the other languages.
+_SUPPORTED_LANGUAGES = {
+    "english": "en",
+}
+
+
+def _clean_and_truncate(text: str, max_length: Optional[int] = None) -> str:
```

Comment:
nit: is `max_length` is more like `max_num_words`?

Reply:
Or `max_num_tokens`.

Reply:
Thanks, I renamed it.